### PR TITLE
Feature/#280 use new relationships implemented in tables when export

### LINKED ISSUE #280: use the new relationships implemented in the tables when exporting the schema

### PR DESCRIPTION
**Pull request by manudous**
Closed #280

### CODE REVIEW

File: src/pods/export/components/relation/components/database-relation-self-export.component.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+          <feMerge>
+            <feMergeNode in="coloredBlur" />
+            <feMergeNode in="SourceGraphic" />
+          </feMerge>
+        </filter>
+      </defs>
+      <path
+        d={getRelationPathBasedOnType(relationType)}
+        fill="none"
+        stroke="#ecad5a"
```
Comment (manudous) [inline src/pods/export/components/relation/components/database-relation-self-export.component.tsx:71]: could we move to a constant becouse we can't use css here but we can use props or variables

File: src/pods/canvas/components/relation/components/clickeable-self.component.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+interface Props {
+  id: GUID;
+  startCoords: Coords;
+  endCoords: Coords;
+  onClick: (relationId: GUID) => void;
+  onDoubleClick: (relationId: GUID) => void;
+}
+
+export const ClickableSelfComponent: React.FC<Props> = props => {
```
Comment (brauliodiez) [inline src/pods/canvas/components/relation/components/clickeable-self.component.tsx:13]: What's this? is this related to export case? I guess you commented you were going to fix something else

File: src/pods/export/components/relation/components/database-relation-self-export.component.tsx (hunk 1 above)
Comment (brauliodiez) [reply src/pods/export/components/relation/components/database-relation-self-export.component.tsx:71]: We are using this: 

```ts
export const exportStylesVariables = {
  PRIMARY_BACKGROUND: '#f1f5f9',
  SECONDARY_BACKGROUND: '#fefdfe',
  ACCENT_COLOR: '#e367f4',
  HIGHLIGHT_COLOR: '#e6eaf2',
  TEXT_COLOR: '#202020',
};
```

but not sure if for this case apply to create a new variable

File: src/pods/canvas/components/relation/components/clickeable-self.component.tsx (hunk 2 above)
Comment (manudous) [reply src/pods/canvas/components/relation/components/clickeable-self.component.tsx:13]: I commented to you, I went to fix this
